### DISCUSSION TITLE
Can "Client Routing" create hidden pages?

### DISCUSSION THREAD

**Discussion by hato810424** [vikejs/vike#1247] [Discussion]
If you try to create a hidden page, the URL will be exposed in `assets/entries/entry-client-routing.[].js`.

Anyone who can read Javascript can read the URL of the hidden page and its contents. Is there any way to hide this?

### Uses,
- To prevent the contents of the administration screen from being seen by anyone other than the administrator.
- To distribute content only to those who know the URL.
- etc..

> Since I'm using translation fanction, my sentences might be strange.

**Comment by brillout** [comment]
Closing in favor of https://github.com/vikejs/vike/issues/1248.
